tests/exporters/onnx/test_export_cli.py

```diff
@@ -730,3 +731,37 @@ def test_complex_synonyms(self):
             model.save_pretrained(tmpdir_in)
 
             main_export(model_name_or_path=tmpdir_in, output=tmpdir_out, task="text-classification")
+
+    @parameterized.expand(_get_models_to_test(PYTORCH_EXPORT_MODELS_TINY_SLIM, library_name="transformers"))
+    def test_exporters_cli_pytorch_with_slim(
+        self,
+        test_name: str,
+        model_type: str,
+        model_name: str,
+        task: str,
+        variant: str,
+        monolith: bool,
+        no_post_process: bool,
+    ):
+        model_kwargs = None
+        if model_type == "speecht5":
+            model_kwargs = {"vocoder": "fxmarty/speecht5-hifigan-tiny"}
+
+        try:
+            self._onnx_export(
+                model_name,
+                task,
+                monolith,
+                no_post_process,
+                slim=True,
+                device="cpu",
+                variant=variant,
+                model_kwargs=model_kwargs,
+            )
+        except NotImplementedError as e:
+            if "Tried to use onnxslim for the model type" in str(e) or "doesn't support the graph optimization" in str(
+                e
+            ):
+                self.skipTest(f"unsupported model type in onnxslim: {model_type}")
+            else:
+                raise e
```

Comment:
they should pass if we're testing them 🤗

```suggestion
        self._onnx_export(
            model_name,
            task,
            monolith,
            no_post_process,
            slim=True,
            device="cpu",
            variant=variant,
            model_kwargs=model_kwargs,
        )
```